src/Product.js

```diff
@@ -1,14 +1,12 @@
 import React from "react";
 import Navbar from "./Components/Navbar/Navbar";
 import "./Product.css";
-import { useLocation, useParams } from "react-router-dom";
+import { useParams } from "react-router-dom";
 import data from "./data.json";
 
 function App2() {
-  const location = useLocation()
   const params = useParams()
   const product = data.find(item =>item.id==params.id)
-  console.log(product)
   return (
     <>
     <Navbar />
```

Comment:
Instead of adding Navbar in all component again and again you can add in index.js outside of the Routes.